### PR TITLE
feat: pricing page

### PR DESCRIPTION
**Pull request by lamartire**

### CODE REVIEW

File: components/PageIntro.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.title {
+  margin-bottom: 37px;
+  font-size: 48px;
+  font-weight: bold;
+  letter-spacing: -0.9px;
+  color: var(--endpass-ui-color-white);
+}
+
+.description p {
+  font-size: 24px;
```
Comment (dudiq) [inline components/PageIntro.vue]: What about defined typographic font sizes? Will we use them or it depends? @kazurus ?

File: components/Pricing/PricingCard.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      </span>
+      <span class="count"> per user<br />identity verified </span>
+    </section>
+    <section class="list">
+      <markered-list>
+        <slot />
+      </markered-list>
+    </section>
+    <section class="button-wrapper">
+      <v-button-link href="#" :skin="buttonSkin" :fluid="true">
```
Comment (dudiq) [inline components/Pricing/PricingCard.vue]: Linting needs to be here =( but maybe later, not in this PR

File: components/VButtonLink.vue [hunk 3]
```diff
@@ -10,6 +10,11 @@ export default {
     skin: {
       type: String,
       default: ""
```
Comment (dudiq) [inline components/VButtonLink.vue]: Maybe define all declared skins here? Like it was in ui/kit/VButton

File: components/PageIntro.vue (hunk 1 above)
Comment (lamartire) [reply components/PageIntro.vue]: At this moment I think we should make all pages by pixel perfect.
Then, when we will have all ready pages – we can define some typography helpers.

File: components/Pricing/PricingCard.vue (hunk 2 above)
Comment (lamartire) [reply components/Pricing/PricingCard.vue]: Not in this PR

File: components/VButtonLink.vue (hunk 3 above)
Comment (lamartire) [reply components/VButtonLink.vue]: Good idea.

**Pull request merged**